cl/cleanup/tests.py

```diff
@@ -293,7 +293,7 @@ def test_tax_court_citation_extractor(self):
                 "2003 T.C. Summary Opinion 150",
             ),
             (
-                """
+                u"""
```

Comment:
I'm curious what inspired this?

Reply:
The root of the issue is that this test case contains ascii characters, so I think Python assumes that the whole thing is ascii encoded. This is a problem because one of the "citation" tokens I scan for is the unicode section marker (https://github.com/mattdahl/courtlistener/blob/issue-435-citation-depth/cl/citations/find_citations.py#L600). So without explicitly declaring *this* test string as a unicode string, I get a `UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 4: ordinal not in range(128)` error when I try to parse it.

Perhaps I am doing something wrong myself; for the life of me I've never really been able to figure out how text encoding really works, and this just seemed like the easiest way to make everything play nicely together.